Code/GraphMol/TautomerQuery/Wrap/rdTautomerQuery.cpp

```diff
@@ -189,7 +224,9 @@ struct TautomerQuery_wrapper {
              python::return_internal_reference<>())
         .def("GetModifiedAtoms", &TautomerQuery::getModifiedAtoms)
         .def("GetModifiedBonds", &TautomerQuery::getModifiedBonds)
-        .def("GetTautomers", getTautomers);
+        .def("GetTautomers", getTautomers)
```

Comment:
Should we add help strings to these no that we're here?

Reply:
Adding docs would definitely be a good thing, but it's unrelated to this one